src/nvim/options.lua

```diff
@@ -2077,6 +2140,25 @@ return {
       abbreviation = 'dip',
       cb = 'did_set_diffopt',
       defaults = { if_true = 'internal,filler,closeoff' },
+      values = {
+        'filler',
+        'context:',
+        'iblank',
+        'icase',
+        'iwhite',
+        'iwhiteall',
+        'iwhiteeol',
+        'horizontal',
+        'vertical',
+        'closeoff',
+        'hiddenoff',
+        'foldcolumn:',
+        'followwrap',
+        'internal',
+        'indent-heuristic',
+        'linematch:',
+        { 'algorithm', { 'myers', 'minimal', 'patience', 'histogram' } },
```

Comment:
The colon after `algorithm` is now missing in the cmdline completion of `:set diffopt=`. The colon needs to be added here, and then something needs to be done in `gen_options.lua` to avoid syntax error in the generated file.

Reply:
Good catch, should be fixed now